tools/c7n_azure/c7n_azure/resources/vm.py

```diff
@@ -23,4 +24,21 @@ class resource_type(object):
         client = 'ComputeManagementClient'
         enum_spec = ('virtual_machines', 'list_all')
         id = 'id'
-        name = 'name'
\ No newline at end of file
+        name = 'name'
+        default_report_fields = (
```

Comment:
yay report fields, i'm hopeful that means custodian report now works here.

Reply:
Yeah, this was one thing that was making us think of augmenting the `resourceGroup` in as a pre-parsed field instead of a custom filter - because it would always be a report field you'd want.